packages/server-wallet/src/chain-service/__chain-test__/chain-service.test.ts

```diff
@@ -415,18 +412,38 @@ describe('concludeAndWithdraw', () => {
   });
 
   it('Successful concludeAndWithdraw with eth allocation', async () => {
-    const {channelId, aAddress, bAddress, state, signatures, outcomeAfter} = await setUpConclude();
-
-    const fingerprintUpdated: FingerprintUpdatedArg = {
+    const {
+      channelId,
+      aAddress,
+      bAddress,
+      state,
+      signatures,
+      newAssetOutcome,
+    } = await setUpConclude();
+
+    const AllocationUpdated: AllocationUpdatedArg = {
       channelId,
-      outcomeBytes: encodeOutcome([outcomeAfter]),
+      asset: zeroAddress,
+      newHoldings: '0x00' as Uint256,
+      externalPayouts: [
+        {
+          amount: BN.from(1),
+          destination: makeDestination(aAddress).toLowerCase(),
+        },
+        {
+          amount: BN.from(3),
+          destination: makeDestination(bAddress).toLowerCase(),
+        },
+      ],
+      internalPayouts: [],
+      newAssetOutcome,
```

Comment:
Might be an improvement to de-duplicate this chunk of code. I'll have a look at it myself now.